admin-client/src/pages/domain/Show.svelte

```diff
@@ -77,6 +84,14 @@
           Provision
         </button>
       {/if}
+      {#if domain.state === 'provisioned'}
```

Comment:
OK, I'm using `canDeprovision` here instead of comparing to `domain.state` before displaying the Deprovision button. I didn't make a corresponding change for the Provision button because the current logic is different: the button is displayed whenever the state is `'pending'` but it will be enabled or disabled based on `canProvision`. In the case of Deprovision the button is only displayed when it's going to be enabled. Does that make sense? If that's what we want then I ought to remove the `disabled={!dnsResults.canDeprovision}` on the Deprovision button because it's never going to be true.